tests/form/robot/ValidationTextBox.html

```diff
@@ -495,13 +495,39 @@
 							doh.robot.typeKeys('123', 1000, 600);
 							doh.robot.sequence(d.getTestCallback(dojo.hitch(this, function(){
 								doh.is('123', this.textbox.focusNode.value, "focusNode.value");
-								doh.is(undefined, this.textbox.get('value'), "get('value')");
+								doh.is(123, this.textbox.get('value'), "get('value')");
 								doh.f(this.textbox.isValid(), "!isValid()");
```

Comment:
OK, the code change looks good to me!

About the tests, I see how the "too few decimal digits" tests waits 1000ms and then checks conditions, whereas the "too few decimal digits, format on blur" test will blur the `<input>`, wait for the _onBlur() event, and then conditions.

Presumably the way to combine those two tests is to first type "123", then do the assertions from the ""too few decimal digits" test, but using `d.getTestErrback()` instead of `d.getTestCallback()`, and then finally continue on as the "too few decimal digits, format on blur" test, i.e. set up the _onBlur() listener and then call `doh.robot.keyPress(dojo.keys.TAB, 1500, {});`.